plugins/modules/keycloak_user.py

```diff
@@ -342,6 +342,10 @@
   description: Representation of the user after module execution.
   returned: on success
   type: dict
+user_created:
+  description: Indicates if a user was created.
+  returned: in success
+  type: bool
```

Comment:
```suggestion
  type: bool
  version_added: 12.0.0
```
Also, can you add a changelog fragment for this change? Thanks.